src/rpc/misc.cpp

```diff
@@ -1313,12 +1440,13 @@ template <class IndexPair>
 void DecorateEasySendTransactionInformation(UniValue& ret, const IndexPair& pair, const size_t& confirmations)
 {
     const auto& key = pair.first;
-    ret.push_back(Pair("found", true));
+
     ret.push_back(Pair("txid", key.txhash.GetHex()));
     ret.push_back(Pair("index", static_cast<int>(key.index)));
-    ret.push_back(Pair("amount", ValueFromAmount(GetAmount(pair.second))));
+    ret.push_back(Pair("amount", !key.invite ? ValueFromAmount(GetAmount(pair.second)) : GetAmount(pair.second)));
```

Comment:
doesn't the value 'pair.second' already have if it's an invite?

Reply:
moved it to key as DecorateEasySendTransactionInformation uses key for polymorphic access to `invite` field in the key for both mempool index and address index
https://github.com/meritlabs/merit/blob/59a402867c267a338f907dc67f4ff8132223f9ea/src/rpc/misc.cpp#L1439
https://github.com/meritlabs/merit/blob/59a402867c267a338f907dc67f4ff8132223f9ea/src/rpc/misc.cpp#L1512